pkg/services/publicdashboards/commands/generate_datasources/generate.go

```diff
@@ -11,6 +11,9 @@ import (
 )
 
 var grafanaDatasources = []string{expr.DatasourceType, "datasource"}
+var unsupportedDataSourcesMap = map[string]bool{
```

Comment:
Can we add a comment here to explain why we are doing this? This should be removed after this [PR](https://github.com/grafana/grafana/issues/74872) is merged